_posts/2023-03-07-vanilla-os-2.0-orchid-initial-work.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Vanilla OS 2.0 Orchid - Initial Work"
+description: "We are happy to announce Vanilla OS 2.0 Orchid! With significant changes that we'd like to go over, like changing the base to Debian Sid, OCI support in ABRoot and more."
+published: true
+layout: article
+comments: true
+---
+
+ 
+## Using Debian Sid as a Base
+
+After discussions and considerations, we decided to **move away from Ubuntu** and **base** our distribution on **Debian Sid.** 
+
+For this reason, we changed the version name and codename, starting with **2.0 Orchid**. We have decided to use **Orchid** as the codename, as [Vanilla derives from Orchids](https://en.wikipedia.org/wiki/Vanilla).
+
+There are a few reasons we switched to Debian Sid:-
+
+1. It is closer to a vanilla experience than in Ubuntu. There were efforts to revert Canonical's opinionated workflow, but it was time-consuming. It forced us to focus on reverting these changes. Ubuntu provides a modified version of the GNOME desktop, which does not match how GNOME envisions its desktop. One of the high-level goals of Vanilla OS is to be as *vanilla* as possible, so we reverted many of these changes to reach that goal.
+
+2. There is no strong opinion on application distributivity. Snap is the primary method to get apps on Ubuntu. Based on our testing and many sources online, there are a lot of issues that Snap hasn't addressed currently, like slow startups, centralization, etc. We prefer to push open and cross-organization efforts, like Flatpak. Our switch to Debian Sid will also address a core issue brought to us by many in our community, that most native applications installed in apx's Ubuntu container are a Snap transitional package, which doesn't work inside the container.
```

Comment:
distributivity -> distribution

Reply:
```suggestion
2. There is no strong opinion on application distribution. Snap is the primary method to get apps on Ubuntu. Based on our testing and many sources online, there are a lot of issues that Snap hasn't addressed currently, like slow startups, centralization, etc. We prefer to push open and cross-organization efforts, like Flatpak. Our switch to Debian Sid will also address a core issue brought to us by many in our community, that most native applications installed in apx's Ubuntu container are a Snap transitional package, which doesn't work inside the container.
```
At last I will create a single commit, will keep this suggestion here.